kilo-code/.kilocode/rules/geo-seo.md

```
@@ -0,0 +1,236 @@
+  <meta property="article:published_time" content="2026-01-15T08:00:00Z">
+  <meta property="article:modified_time" content="2026-03-20T10:30:00Z">
+</head>
+```
+
+Meta description: 150-160 chars, direct factual answer to primary query, one specific statistic, never duplicated across pages. Title: `Primary Keyword — Brand Name`, 50-60 chars, keywords first, unique per page.
+
+## Robots.txt
+
+Allow AI citation crawlers, optionally block AI training crawlers:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)